app/components/cases/cases-header.hbs

```diff
@@ -1,34 +1,40 @@
-{{#if (not isAddingCase)}}
-  <div class="auk-toolbar-complex auk-toolbar-complex--auto">
-    <div class="auk-toolbar-complex__left">
-      <div class="auk-toolbar-complex__item">
-        <h1 class="auk-toolbar__title" data-test-cases-header-title>
-          {{t "cases"}}
-        </h1>
-      </div>
+<div class="auk-toolbar-complex auk-toolbar-complex--auto">
+  <div class="auk-toolbar-complex__left">
+    <div class="auk-toolbar-complex__item">
+      <h1 class="auk-toolbar__title" data-test-cases-header-title>
+        {{t "cases"}}
+      </h1>
     </div>
+  </div>
+  {{#if currentSession.isEditor}}
     <div class="auk-toolbar-complex__right">
       <div class="auk-toolbar-complex__item">
-        {{#if currentSession.isEditor}}
-          <button type="button"
-            data-test-cases-header-add-case
-            class="auk-button auk-button--primary"
-            {{action "toggleAddingCase"}}
-          >
-            <span class="auk-button__content">
-              <div class="auk-icon auk-icon--default ">
-                <i class="ki ki-add"></i>
-              </div>
-              <div class="auk-button__label">
-                {{t "add-case"}}
-              </div>
-            </span>
-          </button>
-        {{/if}}
+        <WebComponents::VlCheckbox 
+          @value={{@showArchived}}
+          @toggle={{@toggleShowArchived}}
+          @label={{t "show-archived" }}
+        />      
+      </div>
+      <div class="auk-toolbar-complex__item">
+        <button type="button"
+          data-test-cases-header-add-case
+          class="auk-button auk-button--primary"
+          {{action "toggleAddingCase"}}
```

Comment:
De `action` helper is oude Ember syntax. Als de function geannoteerd is met `@action` kan hier gewoon `this.toggleAddingCase` gebruikt worden. Als er argumenten meegegeven moeten worden, moet de `fn`-helper gebruikt worden. Vb. `{{fn this.toggleAddingCase case}}`

Deze comment geldt op nog enkele plaatsen in deze PR.

Reply:
Ik geloof dat ze er allemaal uitzijn.
Ik wist oorspronkelijk niet goed wat ik met de dubbele action hier moest aanvangen:
```
{{action attacher.hide}}
{{action "editCase" case}}
```
Maar alles lijkt te werken zonder de `attacher.hide`.